src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -14,7 +20,6 @@
 - Fixed AssertionError in `Series.sort_values` after repr and indexing operations.
 - Fixed UDTF "int64 is not serializable" errors from new Snowflake release in `apply(axis=1)`
 - Fixed binary operations with single-row DataFrame and Series.
-- Fixed incorrect return type in `qcut` when given `Series` input and improved error checking logic.
```

Comment:
why move them in this PR?

Reply:
I can make a new PR to move them if that's preferred, it seemed more convenient to move it in this one

Reply:
Let's do it in a separate one because those changes are irrelevant to this bug fix.